mithril-aggregator/src/commands/serve_command.rs

```diff
@@ -53,6 +53,14 @@ pub struct ServeCommand {
     /// Metrics HTTP server listening port.
     #[clap(long)]
     metrics_server_port: Option<u16>,
+
+    /// Master aggregator endpoint
+    ///
+    /// This is the endpoint of the aggregator that will be used to fetch the latest epoch settings
+    /// and store the signer registrations when the aggregator is running in a slave mode.
+    /// If this is not set, the aggregator will run in a master mode.
+    #[clap(long)]
+    pub master_aggregator_endpoint: Option<String>,
```

Comment:
does this field needs to be `pub` ? I don't even remember why there's some pub fields in this structure.

Reply:
You're right, it's not needed to have it `pub`. I fixed it and all other occurences in the `ServeCommand` :+1: